ioctl/cmd/contract/contractshare.go

```diff
@@ -92,7 +92,7 @@ func isDir(path string) bool {
 
 func isReadOnly(path string) bool {
 	var readOnly = false
-	file, err := os.OpenFile(filepath.Clean(path), os.O_WRONLY, 0666)
+	file, err := os.OpenFile(filepath.Clean(path), os.O_WRONLY, 0600)
```

Comment:
1. we need to clarify what the function is doing. `isReadyOnly` checks whether the file is ready only, if so, why we use `os.O_WRONLY`? Permission error in line 97 could be either no permission to write or no permission to read
2. to my understanding, the last 0666 or 0600 has no impact on the function call. it will be used if `os.O_CREATE` is set

Reply:
Check the write permission before sending it to the editor (https://ide.iotex.io/) where  user can edit the content of the file. After editing finished, the local file will be updated with the editor syncing file.